src/System.Runtime.WindowsRuntime/src/System.Runtime.WindowsRuntime.csproj.disabled

```diff
@@ -130,8 +121,5 @@
     <Compile Include="System\Windows\TokenizerHelper.cs" />
     <Compile Include="System\Windows\UI\Color.cs" />
   </ItemGroup>
-  <ItemGroup>
-    <None Include="project.json" />
-  </ItemGroup>
```

Comment:
I'm expecting we will do a clean-up pass over the projects as part of https://github.com/dotnet/corefx/issues/14584 which can deal with cleaning any of these leftover items.